build.psm1

```diff
@@ -1631,7 +1637,12 @@ function Start-DevPowerShell {
 function Start-TypeGen
 {
     [CmdletBinding()]
-    param()
+    param
+    (
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        $IncFileName
```

Comment:
Maybe we should have 'powershell.inc' as the default value for this parameter, so that `Start-TypeGen` without parameter will continue to work.

Reply:
I can do that if people actually call `Start-TypeGen`. I always use the `-ResGen` swich on `Start-PSBuild` because the reason why I need update ResGen is because I need to build anyway.

Reply:
It's not commonly needed, but when debugging anything about the type catalog, it's handy to just run `Start-TypeGen` without building the rest of the projects. For that purpose, a simple default value like `powershell.inc` would be sufficient, no need to call `New-Options` to figure out the runtime.

Reply:
It's not commonly needed, but when debugging anything about the type catalog, it's handy to just run `Start-TypeGen` without building the rest of the projects. For that purpose, a simple default value like `powershell.inc` would be sufficient, no need to call `New-Options` to figure out the runtime.

Reply:
And a minor comment: the variable name you used in the rest of this function is `incFileName`, maybe it can be changed to `IncFileName`.